app.go

```diff
@@ -60,3 +113,22 @@ func (a *Application) Logger() log.Logger {
 func (a *Application) Statter() stats.Statter {
 	return a.statter
 }
+
+type ApplicationResponse struct {
+	Title  string
+	Text   string
+	Speech string
+	Image  string
+	End    bool
+}
+type Config struct {
```

Comment:
Declare this before it is used. It is annoying to have to scroll down to figure out what is going on.